csrc/lower_validation.cpp

```diff
@@ -306,39 +306,6 @@ class VectorizeValidator : public OptInDispatch {
     }
   }
 
-  // For the producer tensor, it's indexed first by transformed like
```

Comment:
Dead code removed

Reply:
Actually, producer should be validated, https://github.com/NVIDIA/Fuser/pull/298 adds the validation. For this PR, we just do code cleanup.